include/votca/tools/matrix.h

```diff
@@ -17,10 +17,10 @@
 
 #ifndef _VOTCA_TOOLS_MAT_H
 #define _VOTCA_TOOLS_MAT_H
-
 #include "ostream"
-#include "types.h"
-#include "vec.h"
+#include <votca/tools/floatingpointcomparison.h>
+#include <votca/tools/types.h>
+#include <votca/tools/vec.h>
```

Comment:
For headers in the same package, please use relative path (as it was before for vec.h)

Reply:
Is this good practice or just because its been the convention?

Reply:
Convention and consistency with other headers!

Reply:
https://stackoverflow.com/questions/597318/what-are-the-benefits-of-a-relative-path-such-as-include-header-h-for-a-hea

I don't think it matters much, but consistency!